dali/operators/reader/loader/fits_loader.cc

```diff
@@ -77,7 +77,7 @@ void FitsLoader::ReadSample(FitsFileWrapper& target) {
     target.data[output_idx].SetMeta(meta);
 
     // set file path
-    target.filename = std::move(path);
+    target.filename = path;
```

Comment:
What was wrong with the move here?

Reply:
Probably complier would do that for you and it may be considered redundant.

Reply:
Why would it, if there are valid copy and move assignments operators, the one that is selected would depend on the type of the right operand.

Reply:
it's done in a loop, so we can use after move, that was the problem

Reply:
Now that I look at it, I think it should be moved out of the loop. There's a single `target.filename`, but several `target.data[output_idx]`. We don't need to assign `target.filename` several times.